dags/bqetl_subplat.py

```diff
@@ -925,6 +928,19 @@
     mozilla_vpn_derived__all_subscriptions__v1.set_upstream(
         mozilla_vpn_derived__users__v1
     )
+    fivetran_stripe_sync_start = FivetranOperator(
+        connector_id="{{ var.value.fivetran_stripe_connector_id }}",
+        task_id="fivetran_stripe_task",
+    )
```

Comment:
not sure if there's something I need to do here to indicate when the fivetran connector runs relative to this DAG

Reply:
No, the Fivetran connector will get triggered by this task. If you have configured a scheduling interval in the Fivetran UI, then it will automatically get changed saying something like that the connector is manually triggered.